doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -300,26 +297,10 @@ The KVM APIs are documented in Section 4 of the document https://www.kernel.org/
 #### 5.3.3.1. Libvirt Interfaces
 The Libvirt APIs are documented in https://libvirt.org/html/index.html.
 
-<!--
```

Comment:
Please do not delete

Reply:
Please re-introduce Cyborg content:
<!--
### 5.3.4. Cyborg

| **OpenStack Service** | **API Version** |
|-----------------------|-----------------|
| Accelerator: Cyborg   | v2              |

Acceleration Service API: https://docs.openstack.org/api-ref/accelerator/v2/index.html
Please note that the initial version of the [Cyborg API v1.0](https://docs.openstack.org/cyborg/stein/admin/api.html) was deprecated in the OpenStack Train release and will be removed in the Ussuri release.
-->